### ISSUE TITLE
Import of RootCategoryId should be possbile (Magento/Store/Model/Config/Importer/Processor/Create.php)

### ISSUE THREAD

**Issue by larsroettig** [magento/magento2#18956]
### Description (*)

Currently is not possible to define the **root_category_id** in the Project ` config.php`

### Repro 
- Create a config.php like
- Run` bin/magento -n app:config:import`

```php
    'scopes' => [
        'websites' => [
            'admin' => [
                'website_id' => '0',
                'code' => 'admin',
                'name' => 'Admin',
                'sort_order' => '0',
                'default_group_id' => '0',
                'is_default' => '0'
            ],
            'de' => [
                'website_id' => '2',
                'code' => 'de',
                'name' => 'DE',
                'sort_order' => '10',
                'default_group_id' => '2',
                'is_default' => '1'
            ]
        ],
        'groups' => [
            [
                'group_id' => '0',
                'website_id' => '0',
                'code' => 'default',
                'name' => 'Default',
                'root_category_id' => '0',
                'default_store_id' => '0'
            ],
            [
                'group_id' => '2',
                'website_id' => '2',
                'code' => 'de',
                'name' => 'DE',
                'root_category_id' => '2',
                'default_store_id' => '2'
            ]
        ],
        'stores' => [
            'admin' => [
                'store_id' => '0',
                'code' => 'admin',
                'website_id' => '0',
                'group_id' => '0',
                'name' => 'Admin',
                'sort_order' => '0',
                'is_active' => '1'
            ],
            'de_de' => [
                'store_id' => '2',
                'code' => 'de',
                'website_id' => '2',
                'group_id' => '2',
                'name' => ' de DE',
                'sort_order' => '10',
                'is_active' => '1'
            ]
        ]
    ]
```

### Expected behavior (*)
- Import of  **root_category_id** is successfully  

### Benefits
- Simple Store setup for Projects 

### Additional information
In the import Class will the root category set hardcoded to 0 see: 
https://github.com/magento/magento2/blob/fdf16cbdb033f1f11768dc64c57d0e30216e8743/app/code/Magento/Store/Model/Config/Importer/Processor/Create.php#L174-L184

**Comment by magento-engcom-team** [comment]
Hi @larsroettig. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@larsroettig do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @larsroettig. Thank you for your report. 
The issue has been fixed in magento/magento2#18958 by @larsroettig in 2.3-develop branch
Related commit(s):
 - [54f4802d5a9ef58418e395e8d12ab4931907d6eb](https://github.com/magento/magento2/commit/54f4802d5a9ef58418e395e8d12ab4931907d6eb)
 - [144ea18392229aef98c5a21165fba106bd97840d](https://github.com/magento/magento2/commit/144ea18392229aef98c5a21165fba106bd97840d)
 - [0f575bce67a6dbc4fd9226d7b0ef19eaaebe0fed](https://github.com/magento/magento2/commit/0f575bce67a6dbc4fd9226d7b0ef19eaaebe0fed)
 - [b490fb16a0e21eb4b11327d4dcc2e54ebdfdaca4](https://github.com/magento/magento2/commit/b490fb16a0e21eb4b11327d4dcc2e54ebdfdaca4)
 - [803cdb1a15c3f6736c3674a8eb24aa8713992e63](https://github.com/magento/magento2/commit/803cdb1a15c3f6736c3674a8eb24aa8713992e63)

The fix will be available with the upcoming 2.3.1 release.

**Comment by VladimirZaets** [comment]
Hi @larsroettig. Thank you for your report. 
The issue has been fixed in magento/magento2#19237 by @gelanivishal in 2.2-develop branch
Related commit(s):
 - [aa4abec4a587b6054c01c36542e97037cea8b7b1](https://github.com/magento/magento2/commit/aa4abec4a587b6054c01c36542e97037cea8b7b1)
 - [acefdb33b0b060c2080d0cc8c20008288afa33f7](https://github.com/magento/magento2/commit/acefdb33b0b060c2080d0cc8c20008288afa33f7)
 - [5866a39705b817ca6ce49440cbe4d5894ac47063](https://github.com/magento/magento2/commit/5866a39705b817ca6ce49440cbe4d5894ac47063)
 - [fee27127882c0bf2ef832213fef737b259ed5619](https://github.com/magento/magento2/commit/fee27127882c0bf2ef832213fef737b259ed5619)
 - [7be21e2bac3ae2557fcd2ef1e747d0877417944d](https://github.com/magento/magento2/commit/7be21e2bac3ae2557fcd2ef1e747d0877417944d)

The fix will be available with the upcoming 2.2.8 release.
